src/python.ts

```diff
@@ -10,6 +10,9 @@ export const pythonLanguage = LezerLanguage.define({
     props: [
       indentNodeProp.add({
         Body: continuedIndent({except: /^\s*(else|elif|except|finally)\b/}),
+        TupleExpression: delimitedIndent(),
+        DictionaryExpression: delimitedIndent(),
+        ArrayExpression: delimitedIndent(),
```

Comment:
These aren't compiling for me.

I think they're missing a value for `closing`.

https://github.com/codemirror/language/blob/main/src/indent.ts#L264

Compare with:

https://github.com/codemirror/lang-javascript/blob/main/src/javascript.ts#L23

Reply:
Interestingly the `./bin/cm.js build` completes just fine but the demo doesn't work if I switch it to use lang-python.

Reply:
Indeed, that patch never typechecked. 18462e73470c78b should fix it.

Reply:
Apologies for missing that gents. And thanks @marijnh for fixing! 🙌